lesson_10/solid/solid_app/src/main/java/com/codedifferently/lesson10/library/media/Dvd.java

```diff
@@ -0,0 +1,25 @@
+package com.codedifferently.lesson10.library.media;
+
+public class Dvd {
```

Comment:
How are you going to ensure that you can check in or check out a DVD from the library?